libp2p/protocols/pubsub/pubsub.nim

```diff
@@ -11,7 +11,8 @@
 
 import std/[tables, sequtils, sets, strutils]
 import chronos, chronicles, metrics, bearssl
-import ./pubsubpeer,
+import ./errors as pubsub_errors,
```

Comment:
We missed that during review, but `pubsub_errors` should have been exported to limit the backward compat breaking (though is PR is already breaking backward compatibility, so it's not too bad)

Reply:
Fixed in https://github.com/status-im/nim-libp2p/pull/697.